### PR TITLE
Test: increase spring autoconfigure module unit test coverage

### LINKED ISSUE #6199: Increase the spring autoconfigure module unit test coverage
## Why you need it?
Increase the common module unit test coverage

### PR DESCRIPTION
**Pull request by l81893521**
- [ ] I have registered the PR [changes](../changes).

### Ⅱ. Does this pull request fix one issue?
fixes #6199

### CODE REVIEW

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/SeataClientEnvironmentPostProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.core.Ordered;
+
+/**
+ * @description:
+ * @author: zhangjiawei
+ * @date: 2024/1/2
```
Comment (lightClouds917) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/SeataClientEnvironmentPostProcessorTest.java]: remove author and date

**Review by lightClouds917 — CHANGES_REQUESTED** [review]

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SeataPropertiesTest.java
Comment (funky-eyes) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SeataPropertiesTest.java:1]: *PropertiesTest 我希望这些properties的get set的相关方法通过扫描包，解析类，进行反射将其中的method一次性处理掉，至于入参类型可以通过反射拿到参数类型，写一个switch等方式来处理。
*PropertiesTest I want the get set related methods of these properties to scan the package, parse the class, and do reflection to get rid of the method once and for all, and as for the input parameter type can be dealt with by getting the parameter type through reflection, writing a switch, etc.

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/SeataClientEnvironmentPostProcessorTest.java (hunk 1 above)
Comment (l81893521) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/SeataClientEnvironmentPostProcessorTest.java]: fixed

**Review by funky-eyes — COMMENTED** [review]
*PropertiesTest 我希望这些properties的get set的相关方法通过扫描包，解析类，进行反射将其中的method一次性处理掉，至于入参类型可以通过反射拿到参数类型，写一个switch等方式来处理。
*PropertiesTest I want the get set related methods of these properties to scan the package, parse the class, and do reflection to get rid of the method once and for all, and as for the input parameter type can be dealt with by getting the parameter type through reflection, writing a switch, etc.

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SagaAsyncThreadPoolPropertiesTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * limitations under the License.
+ */
+package io.seata.spring.boot.autoconfigure.properties;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @description:
+ * @author: zhangjiawei
```
Comment (lightClouds917) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SagaAsyncThreadPoolPropertiesTest.java:24]: remove author

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SeataPropertiesTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ * limitations under the License.
+ */
+package io.seata.spring.boot.autoconfigure.properties;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @description:
+ * @author: zhangjiawei
```
Comment (lightClouds917) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SeataPropertiesTest.java:24]: remove author

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfigurationTest.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.core.env.PropertiesPropertySource;
+
+import java.util.Properties;
+
+/**
+ * @description:
+ * @author: zhangjiawei
```
Comment (lightClouds917) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/SpringCloudAlibabaConfigurationTest.java:28]: remove author

File: seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/client/LockPropertiesTest.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+ */
+package io.seata.spring.boot.autoconfigure.properties.client;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @description:
+ * @author: zhangjiawei
+ * @date: 2023/12/22
```
Comment (lightClouds917) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-client/src/test/java/io/seata/spring/boot/autoconfigure/properties/client/LockPropertiesTest.java:25]: remove all author

**Review by lightClouds917 — CHANGES_REQUESTED** [review]
remove all author

**Pull request closed without merging**